fw.lua

```diff
@@ -325,7 +304,69 @@ local function _log_event(self, request_client, request_uri, rule, match)
 		end
 	}
 
-	lookup[self._event_log_target](t)
+	lookup[self._event_log_target](log_event)
```

Comment:
this lookup table was moved to a separate module. once you rebase you'll notice the difference